### PR TITLE
[Feat]: Add a demo of Python `sqlalchemy`

### PR DESCRIPTION
**Pull request by 521xueweihan**
## Summary
add sample for  python `sqlalchemy`

- `example.py`：A python script
- `demo`：A minimal example project

### CODE REVIEW

File: python/sqlalchemy/README.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+### 示例代码
+
+以 [example.py](example.py) 为例，仅需替换文件最上方的 OceanBase 连接信息，即可运行示例代码。
+
+```
+# OceanBase 数据库连接参数
+username = 'root' # 「账号管理」页面创建的账号
+password = '' # 「账号管理」页面创建的账号密码
+host = 'localhost' # 「数据库实例」页面开通的公网地址
+port = '3306' # 无需改动
```
Comment (whhe) [inline python/sqlalchemy/README.md:62]: The default port number of OceanBase should be 2881 (observer) or 2883 (obproxy).

File: python/sqlalchemy/demo/db/base.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# -*- coding:utf-8 -*-
+#
+#   Author  :   XueWeiHan
```
Comment (whhe) [inline python/sqlalchemy/demo/db/base.py:4]: We recommend not setting author information in code files, as all changes are already recorded by Git.

**Review by whhe — COMMENTED** [review]
Thanks for your contribution! I left some comments.

**Review by 521xueweihan — COMMENTED** [review]
@whhe rm author and fix port done.

**Comment by whhe** [comment]
Looks like the CI job failed. Could you please add the missing field and fix it?

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -33,6 +33,8 @@ jobs:
             with_oceanbase_container: true
           - name: 'mysql-connector-python'
             language: 'python'
+          - name: 'sqlalchemy'
```
[the remark this answers is not in the document]
Comment (521xueweihan) [reply .github/workflows/ci.yml:37]: fix

**Review by whhe — APPROVED** [review]
LGTM

**Pull request merged**